draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -796,14 +1087,22 @@ Constants: A and B, the parameters of the Weierstrass curve.
 
 Output: (x, y), a point on E.
 
+Sign of y: this encoding does not compute a square root, so there
```

Comment:
I don't think it's necessary (or advisable) to change the sign of y, since the encoding is unambiguous.

Reply:
My observation is that the mini-section "Sign of y: .... "  can be omitted, and make the desired changes in the sign of y internally in the algorithm.

Reply:
The high-level descriptions and straight-line procedures *do* implement the sign changes. So is the suggestion to remove the explanation here?

I'm in favor of explicitly documenting decisions like this (and exceptional cases) rather than making them implicit in the algorithm, since that gives an opportunity to explain the rationale for the decision and documents departures from the algorithms described in the literature.

@armfazh suggests in another comment that these decisions could be documented separately, say, on ePrint. That certainly could work! but to me it seems better to make the document self-contained to the greatest extent possible.

Reply:
Let's keep the full descriptions for the cases were are required.